src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -854,11 +855,21 @@ void style_transfer::import_from_custom_model(variant_map_type model_data,
   const flex_int max_iterations =
       read_opts<flex_int>(model_data, "max_iterations");
   const flex_string model_type = read_opts<flex_string>(model_data, "model");
+  const flex_string _training_time_as_string = read_opts<flex_string>(model_data, "_training_time_as_string");
+  const flex_int training_epochs = read_opts<flex_int>(model_data, "training_epochs");
+  const flex_int training_iterations = read_opts<flex_int>(model_data, "training_iterations");
+  const flex_int num_content_images = read_opts<flex_int>(model_data, "num_content_images");
+  const flex_float training_loss = read_opts<flex_float>(model_data, "training_loss");  
 
   add_or_update_state({{"model", model_type},
                        {"num_styles", num_styles},
                        {"max_iterations", max_iterations},
-                       {"batch_size", DEFAULT_BATCH_SIZE}});
+                       {"batch_size", DEFAULT_BATCH_SIZE},
```

Comment:
Won't this effect a unit test that check what things are in the state? If not, we don't have one so lets' add one

Reply:
Does affect Python unit tests, looking into those right now.